avm/client/src/index.ts

```diff
@@ -82,28 +80,30 @@ class HostImportsConfig {
     }
 }
 
-const interpreter_wasm = toByteArray(wasmBs64);
-
-/// Instantiates WebAssembly runtime with AIR interpreter module
-async function interpreterInstance(cfg: HostImportsConfig, logFunction: LogFunction): Promise<Instance> {
-    /// Create host imports that use module exports internally
+// Instantiates WebAssembly runtime with AIR interpreter module
```

Comment:
Maybe it's better to use three `/` for functions meanind that it's a doc comment?